improver/metadata/amend.py

```diff
@@ -76,10 +77,30 @@ def amend_attributes(cube: Cube, attributes_dict: Dict[str, Any]) -> None:
             Dictionary containing items of the form {attribute_name: value}.
             The "value" item is either the string "remove" or the new value
             of the attribute required.
+            If the new value contains "{}", the existing value will be
+            inserted at this point (no existing value will result in the
+            "{}" being removed, then applied as the attribute value).
+            If the new value contains "{now:.*}", where the .* is a valid
+            date format, then this string is replaced with the current
+            wall-clock time, formatted as specified.
```

Comment:
Perhaps [this docstring](https://github.com/metoppv/improver/blob/master/improver/standardise.py#L214) in `StandardiseMetadata` should be updated to reflect this new functionality?

Reply:
Good idea. Docstring updated.